specification/paths/Shipments.json

```diff
@@ -11,6 +11,12 @@
         "in": "query",
         "type": "string",
         "description": "Comma separated string of the relationship names you want to include the data of.<br> The relationships that can be included are shop, service, contract, service_options, status and files."
+      },
+      {
+        "name": "filter[search]",
+        "in": "query",
+        "type": "string",
+        "description": "Comma separated string of characters you want to look for in the description and recipient_address fields of the shipment."
```

Comment:
I don't think this is a comma separated string of characters is it? Its just a string to search on.